src/components/workspace-utils.js

```diff
@@ -477,8 +478,8 @@ export const RecentlyViewedWorkspaceCard = ({ workspace, submissionStatus, loadi
             loadingSubmissionStats
           }),
           !!googleProject ?
-            h(CloudGcpLogo, { title: 'Google Cloud', role: 'img', style: { marginLeft: 5, height: 16 } }) :
-            h(CloudAzureLogo, { title: 'Microsoft Azure', role: 'img', style: { marginLeft: 5, height: 16 } })
+            h(CloudGcpLogo, { title: cloudProviders.gcp.iconTitle, role: 'img', style: { marginLeft: 5 } }) :
```

Comment:
Setting the height here was causing the Azure one to get truncated:

![image](https://user-images.githubusercontent.com/484484/191799258-54d68c65-1a03-49f0-9563-6518dc51a28b.png)